src/Stratis.Bitcoin.Features.Apps/AppsFeature.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Stratis.Bitcoin.Builder;
+using Stratis.Bitcoin.Builder.Feature;
+using Stratis.Bitcoin.Configuration.Logging;
+using Stratis.Bitcoin.Features.Apps.Interfaces;
+
+namespace Stratis.Bitcoin.Features.Apps
+{
+    public class AppsFeature : FullNodeFeature
```

Comment:
Please comment this and describe the intent of what we are trying todo and achieve

Reply:
OK